services/tenant-ui/frontend/test/components/connections/Invitations.test.ts

```diff
@@ -20,16 +20,17 @@ describe('Invitations', () => {
     wrapper.getComponent({ name: 'DataTable' });
     expect(
       wrapper.findAllComponents({ name: 'CreateConnection' })
-    ).toHaveLength(2);
+    ).toHaveLength(1);
   });
 
   test('table body is rendered with expected values', async () => {
     const wrapper = mountInvitations();
-    const expectedTexts = ['', 'test.alias', 'once'];
+    const expectedTexts = ['', 'test.alias', 'once', 'connections/1.0'];
 
     // td is an expected text or valid date
     wrapper.findAll('tbody td').forEach((td) => {
       const text = td.text();
+      console.log(text);
```

Comment:
Probably left over from checking the test html.

Reply:
Thx! Fixed